src/NuGet.Clients/PackageManagement.UI/Xamls/InfiniteScrollList.xaml.cs

```diff
@@ -196,7 +196,7 @@ private void LoadItems(PackageItemListViewModel selectedPackageItem)
             await NuGetUIThreadHelper.JoinableTaskFactory.RunAsync(async () =>
             {
                 await NuGetUIThreadHelper.JoinableTaskFactory.SwitchToMainThreadAsync();
-                _loadingStatusBar.ItemsLoaded = PackageItems.Count();
+                _loadingStatusBar.ItemsLoaded = currentLoader.State.ItemsCount;
```

Comment:
The last change in this file checks _loader for null. So maybe currentLoader could be null. Can it?